__docs__/wonder-blocks-link/link.argtypes.ts

```diff
@@ -98,6 +110,15 @@ export default {
         },
     },
 
+    startIcon: {
+        control: {type: "select"},
+        description: "Icon to appear before the link label.",
+        options: icons,
+        table: {
+            type: {summary: "IconAsset"},
+        },
+    },
+
```

Comment:
If you want `startIcon` and `endIcon` to be next to each other in the table, you could add a `category` field to `table`. Maybe you could call the category "Icons" or something?